en/tools/config-catalog-generator-cc/data/router/configs.json

```diff
@@ -0,0 +1,203 @@
+{
+  "configs": [
+    {
+      "title": "Router",
+      "options": [
+        {
+          "name": "router",
+          "required": true,
+          "description": "The configurations required for router to route the traffic from different clients to services.",
+          "params": [
+            {
+              "name": "listenerHost",
+              "type": "string",
+              "required": true,
+              "default": "0.0.0.0",
+              "description": "host for Listener of Router."
+            },
+            {
+              "name": "securedListenerHost",
+              "type": "string",
+              "required": true,
+              "default": "0.0.0.0",
+              "description": "Host for secured listener of Router."
+            },
+            {
+              "name": "listenerPort",
+              "type": "string",
+              "required": true,
+              "default": "9090",
+              "description": "Port for listener of Router."
+            },
+            {
+              "name": "securedListenerPort",
+              "type": "string",
+              "required": true,
+              "default": "9095",
+              "description": "Port for secured listener of Router."
+            },
+            {
+              "name": "systemHost",
+              "type": "string",
+              "required": true,
+              "default": "localhost",
+              "description": "The system hostname for system API resources (eg: /testkey and /health)."
+            },
+            {
+              "name": "clusterTimeoutInSeconds",
+              "type": "integer",
+              "required": false,
+              "default": "20",
+              "description": "The timeout for new network connections to hosts in the cluster in seconds."
+            },
+            {
+              "name": "enforcerResponseTimeoutInSeconds",
+              "type": "integer",
+              "required": false,
+              "default": "20",
+              "description": "The timeout for response coming from enforcer to route per API request."
+            }
+          ]
+        }
+      ],
+      "exampleFile": "router.toml"
+    },
+    {
+      "title": "Keystore",
+      "options": [
+        {
+          "name": "keystore",
+          "required": true,
+          "description": "The configurations of key store used in Choreo Connect Adapter",
```

Comment:
This description is the same as the description used for [adapter.keystore]. Shouldn't it be altered as follows: `The configurations of key store used in Choreo Connect Router`?